src/main/java/org/jabref/gui/fieldeditors/contextmenu/ClearField.java

```diff
@@ -0,0 +1,15 @@
+package org.jabref.gui.fieldeditors.contextmenu;
+
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.TextArea;
+
+import org.jabref.logic.l10n.Localization;
+
+class ClearField extends MenuItem {
+
+    public ClearField(TextArea opener) {
+        super(Localization.lang("Clear text"));
+        setOnAction(event -> opener.setText(""));
+    }
+
```

Comment:
Well, the new line character is now at the wrong place 😸. It should be after the last brace and not before (the red "no newline at end of file" marker here at github should disappear. Your IDE probably has an option for adding the newline automatically.